atd-vze/src/Components/GridDateRange.js

```diff
@@ -63,52 +63,62 @@ const GridDateRange = ({ setDateRangeFilter, initStartDate, initEndDate }) => {
   const formatDate = date => moment(date).format("YYYY-MM-DD");
 
   /**
-   * Returns existing selection or date one year ago from today
+   * Returns date one year ago from today
    * @type {Date}
    */
-  const minDate = initStartDate
-    ? parseDate(initStartDate)
-    : new Date(
-        moment()
-          .subtract(1, "year")
-          .format()
-      );
+  const minDate = new Date(
+    moment()
+      .subtract(1, "year")
+      .format()
+  );
```

Comment:
The prettier log shows this variable as unused, should we remove it if not needed?

```
./src/Components/GridDateRange.js
  Line 69:  'minDate' is assigned a value but never used  no-unused-vars
```

Reply:
Thanks for catching this - I removed it before merge.